docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,149 @@
+|1       |88   |8     |SortMergeJoin                                       |11111        |number of output rows               |500000000    |sum       |
+|1       |88   |9     |HashAggregate                                       |22222        |number of output rows               |600000000    |sum       |
+```
+
+Since the two tools are only analyzing Spark event logs they do not have the detail that can be
+captured from a running Spark job.  However it is very convenient because you can run the tools on
+existing logs and do not need a GPU cluster to run the tools.
+
+## 2. Function `explainPotentialGPUPlan` 
+
+### Requirements
+
+- A Spark 3.x CPU cluster
+- The `rapids-4-spark` and `cudf` [jars](../download.md)
+- Ability to modify the existing Spark application code
+
+### How to use
+
+Starting with version 21.12 of the RAPIDS Accelerator, a new function named
+`explainPotentialGPUPlan` is added which can help us understand the potential GPU plan and if there
+are any unsupported features on a CPU cluster.  Basically it can return output which is the same as
+the driver logs with `spark.rapids.sql.explain=all`.
+
+1. In `spark-shell`, add the `rapids-4-spark` and `cudf` jars into --jars option or put them in the
+   Spark classpath.
+
+   For example:
+
+   ```bash
+   spark-shell --jars /PathTo/cudf-<version>.jar,/PathTo/rapids-4-spark_<version>.jar
+   ```
+
+2. Test if the class can be successfully loaded or not.
+
+   ```scala
+   import com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan
+   ```
+
+3. Enable optional RAPIDS Accelerator related parameters based on your setup.
+
+   Enabling optional parameters may allow more operations to run on the GPU but please understand
+   the meaning and risk of above parameters before enabling it. Please refer to [configs
+   doc](../configs.md) for details of RAPIDS Accelerator parameters.
+   
+   For example, if your jobs have `double`, `float` and `decimal` operators together with some Scala
+   UDFs, you can set the following parameters:
+   
+   ```scala
+   spark.conf.set("spark.rapids.sql.incompatibleOps.enabled", true)
+   spark.conf.set("spark.rapids.sql.variableFloatAgg.enabled", true)
+   spark.conf.set("spark.rapids.sql.decimalType.enabled", true)
+   spark.conf.set("spark.rapids.sql.castFloatToDecimal.enabled",true)
+   spark.conf.set("spark.rapids.sql.castDecimalToFloat.enabled",true)
+   spark.conf.set("spark.rapids.sql.udfCompiler.enabled",true)
+   ```
+
+4. Run the function `explainPotentialGPUPlan` on the query DataFrame.
+
+   For example:
+
+   ```scala
+   scala> val df_multi=spark.sql("SELECT value82*value63 FROM df2 union SELECT value82+value63 FROM df2")
+   df_multi: org.apache.spark.sql.DataFrame = [(CAST(value82 AS DECIMAL(9,3)) * CAST(value63 AS DECIMAL(9,3))): decimal(15,5)]
+
+   scala> val output=com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan(df_multi)
+   scala> println(output)
+   ```
```

Comment:
Will this example be out of date in 21.12 with decimal 128 support?

Reply:
yes could be. This is an example based on some un-supported messages.

Reply:
Or should I mention that:
This is an example missing feature based on 21.10 version but it could be implemented in the future release.

Reply:
@sameerz I added a JDBC Scan example instead.